src/InfluxDB/Client.php

```diff
@@ -144,6 +144,44 @@ public function __construct(
     }
 
     /**
+     * @return string
```

Comment:
why not pass $username and $password to the database class? The client class is becoming too public method heavy with this implementation.